reviews/views.py

```diff
@@ -422,6 +422,7 @@ def get_success_url(self):
     def form_valid(self, form):
         "Sets the discontinued continuation on the review"
         review = form.instance
+        # mark_reviewed should already have been done at this point.
```

Comment:
`porposal.mark_reviewed()` is not called when a proposal gets discontinued.

Mark reviewed sets status_review and generates a pdf, which are things I don't think we want to happen to discontinued reviews.

Best to remove this comment to reduce confusion I think. Or am I the one who is confused here?